src/components/global/inputs/radio.tsx

```diff
@@ -16,12 +16,12 @@ const Radio = ({ meta }: { meta: RadioInput }) => {
       <RadioGroup
         defaultValue=""
         value={String(selected)}
-        className="flex flex-row space-x-4"
+        className="flex flex-row space-x-3"
       >
         {options.map((option) => (
           <div
             key={option}
-            className={`flex items-center space-x-2 rounded-sm border py-1 pl-2 pr-4 hover:bg-starlight-blue hover:text-white ${selected === String(option) ? "bg-starlight-blue text-white" : "bg-white"}`}
+            className={`flex items-center space-x-3 rounded-sm border py-3 pl-3 pr-3 hover:bg-starlight-blue hover:text-white ${selected === String(option) ? "bg-starlight-blue text-white" : "bg-white"}`}
```

Comment:
revert these, these are fine.

but you need to add space between title and the radio groups.